external/portable/portable.depproj

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <NuGetDeploySourceItem>Reference</NuGetDeploySourceItem>
+    <NugetRuntimeIdentifier>None</NugetRuntimeIdentifier>
+    <OutputPath>$(RefRootPath)/$(TargetGroup)</OutputPath>
+  </PropertyGroup>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
+  <Target Name="FilterTargetingPackResolvedNugetPackages" />
```

Comment:
Because I don't want any filtering.  Without this I was only getting mscorlib.  Perhaps there's a better way to do this if we update the filtering target but I didn't see a good hook, and I didn't want to list out every assembly.  We might want to do the same for the desktop targeting packs, then move your shim list into shims.proj. /cc @tarekgh